### PR TITLE
Check if version is available in fluvio helm repo before proceeding to install

### LINKED ISSUE #33: CLI: warn if release version is not available for cluster install
Check if version is available in fluvio helm repo before proceeding to install.

### PR DESCRIPTION
**Pull request by nacardin**
Resolves #33

### CODE REVIEW

File: src/cli/src/cluster/install/helm.rs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    debug!("command {:?}", cmd);
+
+    let output = cmd
+        .output()
+        .expect("unable to query for versions of fluvio-core in helm repo");
+
+    debug!("command output {:?}", output);
+
+    serde_json::from_slice(&output.stdout).expect("invalid json from helm search")
```
[the remark this answers is not in the document]
Comment (nacardin) [reply src/cli/src/cluster/install/helm.rs:54]: Seems that in the cli, there are many uses of `expect` for situations where an error is not expected in normal operation. Are we trying to move away from `expect`  all together or are there specific cases where it is ok?

**Comment by nacardin** [comment]
> this PR raise question that we should create helm wrapper crates in future if we need more functionality.

I think we don't use helm often enough to warrant creating and maintaining a full wrapper around it at this time. Perhaps if the wrapper crate downloaded and managed the helm binary, it might be worth it since it saves the operator the trouble of making sure the right version of helm is installed on their system.

File: src/cli/src/cluster/install/helm.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (sehz) [reply src/cli/src/cluster/install/helm.rs:54]: Yes, there are many instances of expect which are unfortunate and we should clean it up.  But those instances are limited to interaction with cmd line api where it's not clear how to recover from error.   This is gray area.  But I guess this is similar so it's ok for now.

**Comment by sehz** [comment]
I agree.  If we are going to create helm wrapper, it should contain helm go library.   There is a question whether to use helm for longer term.  Certainly, there will be some operator that use helm for installation.

**Pull request merged**